.lando.yml

```diff
@@ -57,6 +57,9 @@ tooling:
   phpunit:
     service: appserver
     cmd: /app/vendor/bin/phpunit --configuration web/profiles/custom/az_quickstart/phpunit.xml.dist --bootstrap web/core/tests/bootstrap.php web/profiles/custom/az_quickstart
+  phpunit-fastest:
```

Comment:
Would be good to add a ddev command for this as well.  Our ddev config needs some overall improvement but we've been trying to remain consistent about adding any custom tooling commands to both (see https://github.com/az-digital/az_quickstart/blob/main/.ddev/commands/web/phpunit for an example)

Reply:
Ah, thanks, I've added that, though I haven't been able to test it yet. Getting ddev going has been a bit challenging, mostly for reasons not related to ddev.